front/app/containers/Authentication/steps/BuiltInFields/index.tsx

```diff
@@ -138,6 +141,17 @@ const BuiltInFields = ({
               />
             </Box>
           )}
+          {askEmail && (
+            <Box id="e2e-email-container" mt="16px">
```

Comment:
Minor: Do inputs like email need to be shared components if we are using them in multiple flows?

Reply:
Added it to the refactor ticket: https://www.notion.so/govocal/Refactor-authentication-flows-FE-2917aadc77ba43c7b9e30d9e3716bb06